Source/WineJobDefOf.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Verse;
+using RimWorld;
+
+namespace RwWineMod
+{
+  [DefOf]
+  public static class WineJobDefOf
```

Comment:
It turns out this class is important, It is like a static definition holder for a set of definitions defined in XML. There is a large one in the base game that holds all the jobs, we should probably have one of our own for each Def type that needs it that only holds ours.